### PR TITLE
Tell Eclipse to ignore configurating an M2E connector for com.coveo.fmt-maven-plugin

### PR DESCRIPTION
**Pull request by chanseokoh**
For each plugin in `pom.xml`, there should be an m2e connector in Eclipse that does a proper thing inside the IDE. There is no connector available for `fmt-maven-plugin`, and that's placing an error marker in the IDE. I've been talking to @briandealwis, and we will ignore the plugin for now.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -325,6 +326,20 @@
...
+                  <pluginExecutionFilter>
+                    <groupId>com.coveo</groupId>
+                    <artifactId>fmt-maven-plugin</artifactId>
+                    <versionRange>[0.0,)</versionRange>
+                    <goals>
+                      <goal>check</goal>
+                    </goals>
+                  </pluginExecutionFilter>
+                  <action>
+                    <ignore />
```
Comment (briandealwis) [inline pom.xml]: Would it make sense actually execute the `check` goal?  Could replace this `<action>` clause with:
```
             <action>
               <execute>
                 <runOnIncremental>false</runOnIncremental>
               </execute >
             </action>
```

**Comment by chanseokoh** [comment]
Sounds like `runOnIncremental` could be beneficial. Let's try that. @briandealwis later commits dismiss earlier approvals, so I need your LGTM again.

**Comment by briandealwis** [comment]
Unfortunately the plugin doesn't report which files are in violation (but nor does it on the command-line).  That seems to be a feature added in `fmt-maven-plugin` 2.0.0.
screen shot 2017-10-03 at 2 55 03 pm
screen shot 2017-10-03 at 2 55 18 pm

```
[INFO] --- fmt-maven-plugin:1.8.0:check (default-cli) @ appengine-plugins-core ---
[info] Processed 100 files (0 non-complying).
[error] Found 1 non-complying files, failing build (validateOnly is true)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 4.426 s
[INFO] Finished at: 2017-10-03T14:57:11-04:00
[INFO] Final Memory: 8M/29M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.coveo:fmt-maven-plugin:1.8.0:check (default-cli) on project appengine-plugins-core: Found 1 non-complying files, failing build (validateOnly is true) -> [Help 1]
```

**Comment by chanseokoh** [comment]
Yeah, that seems to be the way it is currently. @loosebazooka can we upgrade it to 2.0.0? (Upgrading is a separate issue from this PR.)

**Comment by briandealwis** [comment]
Meh, the file display isn't captured for display either in 2.0.0.

**Comment by loosebazooka** [comment]
Yeah any newer version is fine

**Pull request merged**